src/SingleProject/Resizetizer/src/AndroidAdaptiveIconGenerator.cs

```diff
@@ -132,8 +150,10 @@ void ProcessAdaptiveIcon(List<ResizedImageInfo> results, DirectoryInfo fullInter
 			Directory.CreateDirectory(dir);
 
 			// Write out the adaptive icon xml drawables
-			File.WriteAllText(adaptiveIconDestination, adaptiveIconXmlStr);
-			File.WriteAllText(adaptiveIconRoundDestination, adaptiveIconXmlStr);
+			if (!File.Exists(adaptiveIconDestination))
+				File.WriteAllText(adaptiveIconDestination, adaptiveIconXmlStr);
+			if (!File.Exists(adaptiveIconRoundDestination))
+				File.WriteAllText(adaptiveIconRoundDestination, adaptiveIconXmlStr);
```

Comment:
This looks fine, because the file contents only change when the app icon name changes.

Reply:
Can we do better and skip the processing as well? Not sure if that makes a difference as it is a string replace.

Reply:
I reworked the code to skip the string replace too